site/content/en/docs/installation/_index.md

```diff
@@ -59,7 +59,8 @@ kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases
 
 ### Add metrics scraping for prometheus-operator
 
-> _Available in Kueue v0.2.1 and later_
+{{< feature-state state="stable" for_version="v0.2.1" >}}
```

Comment:
Let's remove this note, since v0.2 is very old now. I wouldn't expect anyone to be running a version of Kueue older than v0.4